### PR TITLE
github: require release note on pull requests

### PR DESCRIPTION
**Pull request by overvenus**
## What have you changed? (mandatory)

Require release note on pull requests if it should be mentioned.

## What are the type of the changes? (mandatory)

- Misc (other changes)

## How has this PR been tested? (mandatory)

No.

## Does this PR affect documentation (docs) update? (mandatory)

No.

## Does this PR affect tidb-ansible update? (mandatory)

No.

### CODE REVIEW

**Comment by breeswish** [comment]
I think the release note should be changed when release, instead of a single PR?

**Comment by overvenus** [comment]
@breeswish 
How about adding an upcoming section in the changelog?

I think it's okay to change the changelog on PRs, some projects also do this, eg rocksdb. Collecting release note when release is sometimes annoying and we may forget to mention important changes.

**Comment by siddontang** [comment]
PTAL @dcalvin

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -26,9 +26,10 @@ The currently defined types are listed below, please pick one of the types for t
 
 Please describe the tests that you ran to verify your changes. Have you finished unit tests, integration tests, or manual tests? What additional tests would give you greater confidence in this change?
 
-## Does this PR affect documentation (docs) update? (mandatory)
+## Does this PR affect documentation (docs) or release note? (mandatory)
 
-If there is document change, please file a PR in ([docs](https://github.com/tikv/tikv/tree/master/docs)) and add the PR number here.
+- If there is document change, please file a PR in ([docs](https://github.com/tikv/tikv/tree/master/docs)) and add the PR number here.
```
Comment (dcalvin) [inline .github/pull_request_template.md]:
```suggestion
- If there is a document change, please file a PR in ([docs](https://github.com/tikv/tikv/tree/master/docs)) and add the PR number here.
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -26,9 +26,10 @@ The currently defined types are listed below, please pick one of the types for t
...
+- Please update the [release note](https://github.com/tikv/tikv/blob/master/CHANGELOG.md) if this PR should be mentioned.
```
Comment (dcalvin) [inline .github/pull_request_template.md]:
```suggestion
- If this PR should be mentioned in the release note, please update the [release note](https://github.com/tikv/tikv/blob/master/CHANGELOG.md).
```

**Comment by overvenus** [comment]
PTAL, thanks!

**Pull request merged**